introduction/Java_Currency_Formatter/Solution.java

```diff
@@ -0,0 +1,25 @@
+import java.util.Locale;
+import java.util.Locale.Builder;
+import java.text.NumberFormat;
+import java.util.Scanner;
+
+public class Solution {
+  
+  public static final Locale localIndia = new Locale("en", "IN"); 
+  
+  private static String getCurrency(Locale currency, double priceToConvert) {
```

Comment:
it's not currency, it's locale, right?
the other argument can be currencyAmount.
It's not converted, so this name is a bit misleading.